client/apiclient.go

```diff
@@ -18,41 +18,10 @@ type HTTPClient interface {
 	Do(*http.Request) (*http.Response, error)
 }
 
-// TLSConfig contains the parameters needed to configure TLS on the HTTP client
```

Comment:
FYI, the TLS stuff moved toward the bottom of the file. Sorry it makes this file kind of ugly to review.